packages/react/src/components/form/form.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Asterisk,
+  Button,
+  Checkbox,
+  Field,
+  FieldLabel,
+  Form,
+  FormProps,
+  HelperText,
+  Input,
+  Radio,
+  Validation,
+} from '@onfido/castor-react';
+import React from 'react';
+import { Meta, Story } from '../../../../../docs';
+import { FieldsetLegend } from '../fieldset-legend/fieldset-legend';
+import { Fieldset } from '../fieldset/fieldset';
+
+export default {
+  title: 'React/Form',
+  component: Form,
+} as Meta<FormProps<Values>>;
+
+interface Values {
+  firstName: string;
+  lastName: string;
+  email: string;
+  day?: number;
+  month?: number;
+  year?: number;
+  source?: string;
+  agreement: boolean;
+}
+
+export const Playground: Story<FormProps<Values>> = (props) => (
+  <Form {...props}>
+    <Field>
+      <Input name="firstName" required>
+        First name <Asterisk aria-label="required" />
+      </Input>
+      <Validation state="error" if="valueMissing">
+        Please fill in this field
+      </Validation>
+    </Field>
+
+    <Field>
+      <Input name="lastName" required>
+        Last name <Asterisk aria-label="required" />
+      </Input>
+      <Validation state="error" if="valueMissing">
+        Please fill in this field
+      </Validation>
+    </Field>
+
+    <Field>
+      <Input type="email" name="email" required>
+        Email address <Asterisk aria-label="required" />
+      </Input>
+      <Validation state="error" if="valueMissing">
+        Please fill in this field
+      </Validation>
+      <Validation state="error" if="typeMismatch">
+        Please enter a valid email address
+      </Validation>
+    </Field>
+
+    <Field>
+      <FieldLabel htmlFor="dob-day">
+        Date of birth
+        <HelperText>DD / MM / YYYY</HelperText>
+      </FieldLabel>
+      <div
+        style={{
+          display: 'grid',
+          gap: '0.5rem',
+          grid: '1fr / 5rem 5rem 6rem',
+        }}
+      >
+        <Input
+          id="dob-day"
+          type="number"
+          name="day"
+          placeholder="DD"
+          min={1}
+          max={31}
+        />
+        <Input type="number" name="month" placeholder="MM" min={1} max={12} />
+        <Input
+          type="number"
+          name="year"
+          placeholder="YYYY"
+          min={new Date().getUTCFullYear() - 150}
+          max={new Date().getUTCFullYear()}
```

Comment:
[Minor] Probably worth allowing minimum of 18, and a little more realistic maximum...

```suggestion
          min={new Date().getUTCFullYear() - 99}
          max={new Date().getUTCFullYear() - 18}
```

Altough I would very much prefer to live up to 150 years old!

Reply:
Is this a service under-age people can't use? 😂 

The [oldest person to ever live](https://en.wikipedia.org/wiki/Oldest_people) died at 122 years of age.

I was trying to account for medicine improvements in the 21th century 🤣 

I mean, this is just a hypothetical form anyway, but if we're advocating for accessibility 🤷🏻